modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionServiceTest.java

```diff
@@ -235,22 +361,44 @@ private ObjectDefinition _addCustomObjectDefinition(User user)
 					RandomTestUtil.randomString(), StringUtil.randomId())));
 	}
 
+	private ObjectDefinition _addSystemObjectDefinition(
+			long objectFolderId, User user)
+		throws Exception {
+
+		_setUser(user);
+
+		return _objectDefinitionService.addSystemObjectDefinition(
+			RandomTestUtil.randomString(), user.getUserId(), objectFolderId,
+			false,
+			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+			"Test", null, null,
```

Comment:
I followed the same pattern that i saw in the several tests. But ok, I will change.

Reply:
@seltong I'm keeping "Test" because [this](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/definition/util/ObjectDefinitionUtil.java#L33-L35).